tests/test_config.py

```diff
@@ -328,13 +339,50 @@ def test_invalid_enviroment_variables_config(monkeypatch, capsys):
         _, err = capsys.readouterr()
         assert  "error: unrecognized arguments: --foo" in err
 
+
 def test_cli_overrides_enviroment_variables_module(monkeypatch):
     monkeypatch.setenv("GUNICORN_CMD_ARGS", "--workers=4")
     with AltArgs(["prog_name", "-c", cfg_file(), "--workers", "3"]):
         app = NoConfigApp()
     assert app.cfg.workers == 3
 
 
+def test_wsgi_app_config(capsys):
```

Comment:
To avoid boilerplate, we could use `pytest.mark.parametrize`. See `test_reload` for an example.